cms/jinja2/templates/pages/index_page.html

```diff
@@ -0,0 +1,40 @@
+{% extends "templates/base_page.html" %}
+{% from "components/document-list/_macro.njk" import onsDocumentList %}
+{% from "components/related-content/_macro.njk" import onsRelatedContent %}
+
+{% block main %}
+
+    {% if page.description %}
+        <p>{{ page.description }}</p>
+    {% endif %}
+
+    {% if formatted_featured_pages %}
+        {# fmt:off #}
+        {{
+            onsDocumentList({
+                    "documents": formatted_featured_pages
+                })
+        }}
+        {# fmt:on #}
```

Comment:
```suggestion
        {{ onsDocumentList({"documents": formatted_featured_pages}) }}
```

Reply:
Done in [First round of renaming and refactoring (updated the migration file)](https://github.com/ONSdigital/dis-wagtail/pull/78/commits/19a565709a37e4d47e83b51f46f0af0de718413a)